Adyen/Model/Checkout/CreateCheckoutSessionRequest.cs

```diff
@@ -249,12 +249,28 @@ public CreateCheckoutSessionRequest(AccountInfo accountInfo = default(AccountInf
             this.ShopperReference = shopperReference;
             this.ShopperStatement = shopperStatement;
             this.SocialSecurityNumber = socialSecurityNumber;
-            this.SplitCardFundingSources = splitCardFundingSources;
+            // use default value if no "splitCardFundingSources" provided
+            if (splitCardFundingSources == null)
```

Comment:
This change should be in the Checkout PR. Or not even there if it didn't came from the generator.